Dockerfile.run

```diff
@@ -1,13 +1,15 @@
+FROM alpine:3.4
 
-FROM    alpine:3.4
-RUN     apk -U add \
-            python \
-            py-pip
 
-COPY    requirements.txt /code/requirements.txt
-RUN     pip install -r /code/requirements.txt
+COPY dist/docker-compose-Linux-x86_64 /usr/local/bin/docker-compose
```

Comment:
Please move this after the `RUN` so that we can use the build cache for the `RUN` layer.